app/code/Magento/Ui/view/base/web/js/grid/controls/columns.js

```diff
@@ -80,8 +80,8 @@ define([
 
         /**
          * Helper, checks
-         *  - if less than one item choosen
-         *  - if more then viewportMaxSize choosen
+         *  - if less than one item chosen
+         *  - if more then viewportMaxSize chosen
```

Comment:
if more than*

Reply:
Hello @danmooney 
Thank you for the suggestion, I have update PR accordingly.
Can you review it?

Reply:
Yes it’s fine but please proofread your commit messages as well.  I recommend rebasing to remove commit with misspellings.